Documentation/Contributing/ExperimentalFeatures.md

```diff
@@ -56,7 +56,7 @@ public static void MyCommand()
 Adding a component menu:
 
 ```c#
-[AddComponentMenu("MRTK/Experimental/MyCommand")]
+[AddComponentMenu("Mixed Reality Toolkit/Experimental/MyCommand")]
```

Comment:
This actually contradicts with coding guidelines page. Cntrl+F "AddComponentMenu" in this PR
One uses "MRTK/blah" while this uses "Mixed Reality Toolkit/blah"

Reply:
will restore

Reply:
done